### PR TITLE
Gitpod Integration

### PR DESCRIPTION
**Pull request by sean-e-dietrich**
Adds [Gitpod](https://gitpod.io) Integration

### CODE REVIEW

File: docs/content/use-cases/gitpod.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Usage with Gitpod"
+---
+
+## Gitpod
+
+[Gitpod](https://www.gitpod.io/) is an only development environment.
```
Comment (shelane) [inline docs/content/use-cases/gitpod.md:7]: Consider "Gitpod is a browser-based development environment."
or
"Gitpod is a browser only development environment."

But, it probably needs a bit more explanation.

File: docs/content/use-cases/gitpod.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Getting Started
+
+To get started create a `.gitpod.yml` file within your project.
```
Comment (shelane) [inline docs/content/use-cases/gitpod.md:11]: Add comma:
To get started, create a ...

File: docs/content/use-cases/gitpod.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+```yaml
+tasks:
+  - before: bash <(curl -fsSL https://get.docksal.io)
+    init: fin init
+```
+
+Once created you are ready to start your project within the [Gitpod Workspace](https://www.gitpod.io/docs/getting-started).
```
Comment (shelane) [inline docs/content/use-cases/gitpod.md:19]: Add comma:
Once created, you are ready...

File: docs/content/use-cases/gitpod.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+For more information on creating the Gitpod Configuration file. Consult the [Gitpod Getting Started](https://www.gitpod.io/docs/getting-started)
```
Comment (shelane) [inline docs/content/use-cases/gitpod.md:21]: Change period to comma:
... on creating the Gitpod configuration file, consult the...

**Review by shelane — CHANGES_REQUESTED** [review]
Just getting a head start on the review ;-)

**Review by shelane — APPROVED** [review]
Docs look good.

**Review by lmakarov — APPROVED** [review]

**Pull request merged**